toil_orchestrator/models.py

```diff
@@ -10,6 +10,7 @@ class Status(IntEnum):
     RUNNING = 2
     COMPLETED = 3
     FAILED = 4
+    UNKOWN = 5
```

Comment:
`UNKOWN`? @nikhil

Reply:
This is for when the leader job status show it is finished or exited but the child jobs cannot be determined